documentation/API_documentation/Connectivity_API.md

```diff
@@ -0,0 +1,82 @@
+|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **HTTP Request**<br> POST \<base-url>/connectivity/v0/status<br>**QueryParameters**<br> No query parameters are defined.<br>**Path Parameters**<br> No path parameters are defined.<br>**Request Body Parameters**<br> **ueId**: UE identifier object, contains 4 different identifiers, at least 1 has to be set<br> **uePort (optional):** User equipment port. Device port may be required along with IP address to identify the target device <br> **eventType:** enum type value, for now only one is supported: UE_ROAMING_STATUS<br><br>**Response**<br> **200: Connectivity status checked**<br>  Response body:<br> **ueId**: UE identifier object, contains 4 different identifiers, at least 1 has to be set<br> **uePort (optional):** User equipment port. Device port may be required along with IP address to identify the target device <br> **eventType:** enum type value, contains one of following: UE_REACHABLE, UE_CONNECTION_LOST, UE_ROAMING_STATUS <br> **eventStatus:** enum type value, contains on of the following: ROAMING_ON, ROAMING_OFF <br><br> **400:** **Invalid input.**<br> **401:** **Un-authorized, missing or incorrect authentication.**<br> **405:** **Invalid input**<br> **500:** **Session not created**<br> **503:** **Service temporarily unavailable** |
+<br>
```

Comment:
Only UE_ROAMING_STATUS is available
Response codes: 200, 400, 401, 403, 404, 500, 503